__snapshots__/client/components/Sidebar/Sidebar.test.tsx.snap

```diff
@@ -378,6 +378,24 @@ exports[`Test Sidebar Sidebar renders correctly 1`] = `
             </button>
           </a>
         </li>
+        <li>
```

Comment:
tbh might as well delete all snapshot tests, they're useless and bloat diffs

Reply:
We use them properly now (the classNames correlate to static CSS, not a random hashed string that changes every time) so they're as good as snapshot tests will ever be.

Reply:
I am in favor of deleting the rest of them in #779